shardingsphere-test/shardingsphere-integration-test/shardingsphere-integration-test-scaling/src/test/java/org/apache/shardingsphere/integration/data/pipeline/cases/general/PostgreSQLMigrationGeneralIT.java

```diff
@@ -70,36 +68,39 @@ public static Collection<ScalingParameterized> getParameters() {
         createScalingRule();
-        createSchema("test");
-        createOrderTableRule();
-        createOrderItemTableRule();
-        createOrderTable();
-        createOrderItemTable();
-        // TODO wait kernel support create index if not exists
-        // createTableIndexList("test");
-        executeWithLog("COMMENT ON COLUMN test.t_order.user_id IS 'user id';");
+        createSourceSchema("test");
+        createSourceOrderTable();
+        createSourceOrderItemTable();
+        createSourceTableIndexList("test");
+        createSourceCommentOnList("test");
+        addSourceResource();
+        addTargetResource();
+        createTargetOrderTableRule();
+        createTargetOrderItemTableRule();
         SnowflakeKeyGenerateAlgorithm keyGenerateAlgorithm = new SnowflakeKeyGenerateAlgorithm();
         Pair<List<Object[]>, List<Object[]>> dataPair = ScalingCaseHelper.generateFullInsertData(keyGenerateAlgorithm, parameterized.getDatabaseType(), TABLE_INIT_ROW_COUNT);
-        getJdbcTemplate().batchUpdate(getExtraSQLCommand().getFullInsertOrder(), dataPair.getLeft());
-        getJdbcTemplate().batchUpdate(getExtraSQLCommand().getFullInsertOrderItem(), dataPair.getRight());
-        addTargetResource();
-        startIncrementTask(new PostgreSQLIncrementTask(getJdbcTemplate(), new SnowflakeKeyGenerateAlgorithm(), "test", true, 20));
-        executeWithLog(getCommonSQLCommand().getAlterOrderWithItemAutoTableRule());
-        String jobId = getScalingJobId();
-        waitScalingFinished(jobId);
-        stopScaling(jobId);
-        executeWithLog(String.format("INSERT INTO test.t_order (id,order_id,user_id,status) VALUES (%s, %s, %s, '%s')", keyGenerateAlgorithm.generateKey(), System.currentTimeMillis(),
-                1, "afterStopScaling"));
-        startScaling(jobId);
+        JdbcTemplate jdbcTemplate = new JdbcTemplate(getSourceDataSource());
+        jdbcTemplate.batchUpdate(getExtraSQLCommand().getFullInsertOrder(), dataPair.getLeft());
+        jdbcTemplate.batchUpdate(getExtraSQLCommand().getFullInsertOrderItem(), dataPair.getRight());
+        checkOrderMigration(jdbcTemplate);
+        checkOrderItemMigration();
+    }
+    
+    private void checkOrderMigration(final JdbcTemplate jdbcTemplate) {
+        startMigrationOrder();
+        startIncrementTask(new PostgreSQLIncrementTask(jdbcTemplate, "test", false, 20));
+        String jobId = getJobIdByTableName("t_order");
+        waitMigrationFinished(jobId);
         assertCheckScalingSuccess(jobId);
         assertGreaterThanInitTableInitRows(TABLE_INIT_ROW_COUNT, "test");
-        applyScaling(jobId);
-        assertPreviewTableSuccess("t_order", Arrays.asList("ds_2", "ds_3", "ds_4"));
-        assertPreviewTableSuccess("t_order_item", Arrays.asList("ds_2", "ds_3", "ds_4"));
+    }
+    
+    private void checkOrderItemMigration() {
+        startMigrationOrderItem();
+        String jobId = getJobIdByTableName("t_order");
+        waitMigrationFinished(jobId);
```

Comment:
Looks `getJobIdByTableName("t_order")` should be `getJobIdByTableName("t_order_item")`